extension/framework/runner/environment.go

```diff
@@ -254,19 +254,33 @@ func (env *Environment) getTransaction(isNew bool, options *transaction.TxOption
 	env.dbTransactions = append(env.dbTransactions, tx)
 	return tx, nil
 }
-
+func getSpecifiedFunction(env *Environment, functionName string, setValue otto.Value) func(call otto.FunctionCall) otto.Value {
```

Comment:
nit: should include white space between functions.

Reply:
Added